lib/internal/test_runner/reporter/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const symbols = {
+  '__proto__': null,
```

Comment:
Nitpick: quotes aren't needed here

Reply:
![image](https://github.com/nodejs/node/assets/52945500/39c5d81b-5eac-42db-8497-9e3d910bab86)

Reply:
If that's the case, then ignore my suggestion, sorry.